components/infobox/wikis/formula1/infobox_team_custom.lua

```diff
@@ -0,0 +1,117 @@
+---
+-- @Liquipedia
+-- wiki=formula1
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local TeamTemplates = require('Module:Team')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+local Center = Widgets.Center
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+local Chronology = Widgets.Chronology
+
+local _args
+local _team
+
+local STATISTICS = {
+	{name = 'races', name = 'Races'},
+	{name = 'wins', name = 'Wins'},
+	{name = 'podiums', name = 'Podiums'},
+	{name = 'poles', name = 'Pole positions'},
+	{name = 'fastestlaps', name = 'Fastest Laps'},
+	{name = 'points', name = 'Career Points'},
+	{name = 'firstentry', name = 'First entry'},
+	{name = 'firstwin', name = 'First win'},
+	{name = 'lastentry', name = 'Last entry'},
+}
+
+---@param frame Frame
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	_args = _team.args
+
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.getWikiCategories = CustomTeam.getWikiCategories
+	return team:createInfobox()
+end
+
+---@return WidgetInjector
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:addCustomCells(widgets)
+	Array.extendWith(widgets, CustomTeam._statisticsCells(_args))
+
+	if _args.academy then
+		local academyTeams = Array.map(_team:getAllArgsForBase(_args, 'academy'), function(team)
+			return TeamTemplates.team(nil, team)
+		end)
+		Array.extendWith(widgets,
+			{Title{name = 'Academy Team' .. (Table.size(academyTeams) > 1 and 's' or '')}},
+			Array.map(academyTeams, function(academyTeam) return Center{content = {academyTeam}} end)
+		)
+	end
```

Comment:
Seems unused
```suggestion
```

Reply:
as i understand it they want to add it to use it

Reply:
Re-added, It is something we planned to have it (we'll be using this as well)